victor_hardware_interface/scripts/arm_command_gui.py

```diff
@@ -185,22 +185,22 @@ def __init__(self, parent_layout, arm_name, box_row):
         self.groupbox.setFlat(False)
         parent_layout.addWidget(self.groupbox, box_row, 0)
 
-        print 'Getting', self.name, 'current joint values ...',
+        print('Getting', self.name, 'current joint values ...')
         sys.stdout.flush()
         while (not self.arm_status_updated) or (not self.gripper_status_updated):
             time.sleep(0.01)
-        print 'Done'
+        print('Done')
 
         self.reset_arm_slider_to_current_value()
         self.reset_gripper_position_slider_to_current_value()
 
-        print 'Getting', self.name, 'current control mode ...',
+        print('Getting {} current control mode ...'.format(self.name))
```

Comment:
I'd use f-strings

    print(f'Getting {self.name} current control mode')

Reply:
How do you get f-strings to work in python2?

Reply:
oh fair enough, you probably can't. Nevermind then.